specification/monitor/data-plane/preview/2018-09-01-preview/MetricsPost_API.json

```diff
@@ -21,8 +21,11 @@
   "paths": {
     "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProvider}/{resourceTypeName}/{resourceName}/metrics": {
       "post": {
-        "summary": "Ingestion API used by Azure custom metrics\r\n            Responsible of below operation\r\n            1. Authorize the request\r\n            2. Deserialize payload request body\r\n            3. Perform validation of the payload\r\n            4. Commits the payload for metrics ingestion",
-        "operationId": "azureMonitorCustomMetricsIngestionApi",
+        "tags": [
+          "Metrics"
+        ],
+        "operationId": "Metrics_POST",
+        "description": "**Post the metric values for a resource**.",
```

Comment:
I would suggest something a bit more descriptive, at the least remove the asterisks.

Reply:
I am trying to have consistent formatting similar to our metrics List documenetation here => https://review.docs.microsoft.com/en-us/rest/api/monitor/metrics/list?branch=mdm

Reply:
Oh I see this is markdown, makes sense.